client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -96,6 +88,9 @@ const ClinicalDetails: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Element
     const saveForm = (e: any) => {
         e.preventDefault();
         const values = getValues();
+        if(values.isSymptomsStartDateUnknown) {
+            values.symptomsStartDate = null;
```

Comment:
Instead of settings the `symptomsStartDate` to null before saving the form you should call the `setValue` function supplied by `react-hook-form` when you change the `isSymptomsStartDateUnknown` to true. If you look in the ClinicalDetails file you can see examples of this between lines 115-149.